# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1486886972 -3600
#      Sun Feb 12 09:09:32 2017 +0100
# Branch stable
# Node ID 6da3526ccf334dc97cf8b06b508a28898aaeb41d
# Parent  039dfdc2d2ada3f24bdfa9154e848c98bf8fedb2
fix settings update for existing editor tabs (regression from b3ced5e3cebb)

* file_editor (add_file_editor_tab): connect signal settings_changed with
  the correct slot notice_settings

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -2010,17 +2010,17 @@ file_editor::add_file_editor_tab (file_e
   connect (f, SIGNAL (edit_mfile_request (const QString&, const QString&,
                                           const QString&, int)),
            this, SLOT (handle_edit_mfile_request (const QString&,
                                                   const QString&,
                                                   const QString&, int)));
 
   // Signals from the file_editor non-trivial operations
   connect (this, SIGNAL (fetab_settings_changed (const QSettings *)),
-           f, SLOT (settings_changed (const QSettings *)));
+           f, SLOT (notice_settings (const QSettings *)));
 
   connect (this, SIGNAL (fetab_change_request (const QWidget*)),
            f, SLOT (change_editor_state (const QWidget*)));
 
   connect (this, SIGNAL (fetab_file_name_query (const QWidget*)),
            f, SLOT (file_name_query (const QWidget*)));
 
   connect (this, SIGNAL (fetab_save_file (const QWidget*, const QString&,
